app/graphql/mutations/create_need.rb

```diff
@@ -9,11 +9,12 @@ class Mutations::CreateNeed < Mutations::BaseMutation
   argument :state, String, required: false
   argument :zip_code, String, required: true
   argument :supporters_needed, Integer, required: true
+  argument :categories, [ID], required: false
```

Comment:
Good to confirm it's not required & users can view all needs if they want from a FE perspective